src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      transition-duration: 300ms;
```

Comment:
when I left cursor from a card, the card sharp jumping to default size 
please check it

Reply:
I don't have that problem. everything works